previews/azure-functions-storage-queues.md

```diff
@@ -42,7 +42,8 @@ When using Azure Functions with Azure Storage Queues, the following points must
 - When using the default recoverability or specifying custom number of immediate retries, the number of delivery attempts specified on the underlying queue or Azure Functions host must be more than then number of the immediate retries. The Azure Functions default is 5 (`DequeueCount`) for the Azure Storage Queues trigger.
 - Delayed Retries are not supported.
 - Saga timeouts are not supported.
+- Delayed messages sent to Function endpoints are not supported.
```

Comment:
Now that I'm thinking about it, it's not that delayed messages are not supported. They will not always work as expected. The receiving endpoint will get the delayed messages, but only when it's given a chance to run. Otherwise, delayed messages might not be processed for far longer than they should be delayed.

Example: the endpoint A sends delayed messages to the endpoint B. The endpoint B doesn't process anything for a while and is offloaded by the Functions environment. The messages for B should be processed by the won't until there's a regular (not delayed) message that rehydrates the endpoint B. At that point, all the delayed messages will be processed.

Reply:
I just thought it was better to say it was unsupported. If we do not already have an issue, perhaps we should raise one and link to it? In there we can capture the details of what it would look like. That also gives the benefit of a developer trying to figure out what went wrong finding the issue and the details.

Reply:
I've [raised an issue](https://github.com/Particular/NServiceBus.AzureFunctions.StorageQueues/issues/10). I would still want to articulate that the features will not work _as expected_ in certain scenarios. There are workarounds (listed in the issue).

Reply:
The text currently says 
> Saga timeouts and delayed messages sent to Function endpoints are not guaranteed to execute as expected.

and the issue is raised, so I'm good with that.